src/components/info-sheet/info-sheet.html

```diff
@@ -1,17 +1,68 @@
 <action-sheet>
   <div class="info-container" [ngSwitch]="sheetType">
-    <info-sheet-template *ngSwitchCase="'address-copied'" type="success">
+    <info-sheet-template *ngSwitchCase="'address-copied'" type="success" option="false">
       <span sheet-title-text translate>Copied {{params.coin.toUpperCase()}} address</span>
       <span sheet-text class="text-address" translate>{{params.address}}</span>
       <span sheet-button-text translate>OK</span>
     </info-sheet-template>
-    <info-sheet-template *ngSwitchCase="'receiving-bitcoin'" type="info">
+
+    <info-sheet-template *ngSwitchCase="'receiving-bitcoin'" type="info" option="false">
       <span sheet-title-text translate>Receiving {{params.coinName}} address</span>
       <span sheet-text translate>
         Your wallet address is similar to a bank account number. You can add funds to your wallet by withdrawing funds you may have
         stored on an exchange to your wallet address. Or other {{params.coinName}} users can send you money by scanning your
         QR code.</span>
       <span sheet-button-text translate>GOT IT</span>
     </info-sheet-template>
+
+    <info-sheet-template *ngSwitchCase="'paper-key-unverified'" type="warning" option="true">
```

Comment:
seems like the option input is not necessary since the primary button can always emit true and the secondary button can always emit false.

Reply:
You're right. At first, I thought it could be useful but I was wrong. Thanks.